charts/metallb/templates/rbac.yaml

```diff
@@ -0,0 +1,161 @@
+{{- if .Values.rbac.create -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "metallb.fullname" . }}:controller
+  labels:
+    {{- include "metallb.labels" . | nindent 4 }}
+rules:
+- apiGroups: [""]
+  resources: ["services"]
+  verbs: ["get", "list", "watch", "update"]
+- apiGroups: [""]
+  resources: ["services/status"]
+  verbs: ["update"]
+- apiGroups: [""]
+  resources: ["events"]
+  verbs: ["create", "patch"]
+{{- if .Values.psp.create }}
+- apiGroups: ["policy"]
+  resources: ["podsecuritypolicies"]
+  resourceNames: [{{ printf "%s-controller" (include "metallb.fullname" .) | quote}}]
```

Comment:
I see you use printf here and template later, I prefer template, it's easier to read

Reply:
`["{{ template "metallb.fullname" . }}-controller"]`

Reply:
Thanks for spotting the inconsistency, I'll fix it.

Reply:
Fixed.

Reply:
Fixed